src/collectd.conf.pod

```diff
@@ -304,19 +298,18 @@ metrics. For servers it is recommended to set this to a non-zero value, though.
 
 You can set the limits using B<WriteQueueLimitHigh> and B<WriteQueueLimitLow>.
 Each of them takes a numerical argument which is the number of metrics in the
-queue. If there are I<HighNum> metrics in the queue, any new metrics I<will> be
-dropped. If there are less than I<LowNum> metrics in the queue, all new metrics
-I<will> be enqueued. If the number of metrics currently in the queue is between
-I<LowNum> and I<HighNum>, the metric is dropped with a probability that is
-proportional to the number of metrics in the queue (i.e. it increases linearly
-until it reaches 100%.)
+queue. If there are I<HighNum> metrics in the queue a random element from the
+queue will be dropped. This results in the data becoming more and more sparse
+without dropping a lot of continguous values. An exception from the random
+selection of elements to drop are the I<LowNum> most recent elements which will
+never be dropped. This can be used to ensure that one miss-behaving write
+plugin does not result in dropped values for other well-behaved write plugins.
```

Comment:
Alternative phrasing:
> If there are `HighNum` metrics in the queue a random element from the queue will be dropped. An exception from the random selection of elements to drop are the `LowNum` most recent elements which will never be dropped.  Spreading the loss over the `LowNum` - `HighNum` range avoids large time gaps in the data on write hiccups. This way one miss-behaving write plugin is less likely to result in dropped values for well-behaved write plugins.

Reply:
Sounds good to me, so I copied more or less verbatim into a new commit.
Thanks

Reply:
I thought the original "does not result in dropped values" sounded a bit too strong based on Matthias comments above. :-)

Reply:
You can resolve this now, I do not seem to have rights for that.